### ISSUE TITLE
Build tools

### ISSUE THREAD

**Issue by NouberNou** [NouberNou/intercept#6]
Need a robust build/packaging environment.

@Glowbal and @jaynus I am sure can give some insight on how to go about this or if they want to implement an existing one (Carma2 one might be easy enough?).

It just needs to compile the main intercept host DLL, and then package up everything in the client directory as distributed source includes, and then build and pack up the PBO. Super simple.

**Comment by Glowbal** [comment]
Carma2 build environment is ok. I think we will want to set up 3 build scripts;

- extensions (including the intercept dll)
- PBOs
- both extensions and PBOs (Which can just call the first two build scripts)

**Comment by dedmen** [comment]
We have extensions ready. Only thing missing is https://github.com/intercept/intercept/issues/167.
Should we continue tracking that in #167 and close this?

**Comment by jonpas** [comment]
We don't have a script for a full build, just parts, so I'd leave that open.

**Comment by dedmen** [comment]
Nope. But cmake does most of what one needs, the pbo's are the same as on windows anyway.
